pyzabbix/sender.py

```diff
@@ -385,3 +449,57 @@ def send(self, metrics):
         for m in range(0, len(metrics), self.chunk_size):
             result.parse(self._chunk_send(metrics[m:m + self.chunk_size]))
         return result
+
+    def _create_active_checks_request(self, host):
+        """Create a formatted request to zabbix from a host name.
+
+        :type host: str
+        :param host: The host to ask about
+
+        :rtype: str
+        :return: Formatted zabbix request
+        """
+        request = '{"request":"active checks","host":"%s"}' % host
+        request = request.encode("utf-8")
+        logger.debug('Request: %s', request)
+        return request
+
+    def send_active_checks(self, host):
+        """Get active checks from the zabbix server.
+
+        :type host: str
+        :param host: host to ask about
+
+        :rtype: :class:`pyzabbix.sender.ZabbixActiveChecksResponse`
+        :return: Parsed response from Zabbix Server
+        """
+        request = self._create_active_checks_request(host)
+        packet = self._create_packet(request)
+
+        for host_addr in self.zabbix_uri:
```

Comment:
I think this part must be refactored as a function `_send` because you duplicating big chunk code of `_chunk_send` here.